backend/src/test/java/com/google/models/AnomalyTest.java

```diff
@@ -0,0 +1,146 @@
+package com.google.models;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+/** Contain tests for methods in {@link Anomaly} class. */
+@RunWith(JUnit4.class)
+public final class AnomalyTest {
+  private static final int TIMESTAMP_CONSTANT = 1;
+  private static final String METRIC_NAME = "Sample metric name";
+  private static final String DIMENSION_NAME = "Sample dimension name";
+  private static final Map<Timestamp, MetricValue> DATA_POINTS = ImmutableMap.of( 
+      Timestamp.getDummyTimestamp(1), new MetricValue(1), 
+      Timestamp.getDummyTimestamp(2), new MetricValue(2), 
+      Timestamp.getDummyTimestamp(3), new MetricValue(3));
+  private static final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  private static final Anomaly anomaly = new Anomaly(Timestamp.getDummyTimestamp(TIMESTAMP_CONSTANT), 
```

Comment:
you can call it `private static final Anomaly ANOMALY` to make it clear as what you have explained to catherine about it not beng changed throughout any of the unit tests